src/js/utils.js

```diff
@@ -82,6 +82,26 @@ export function isURL(value) {
   return urlRegexp.test(value)
 }
 
+const linkGroup = /<([\w\-.~!*'();:@&=+$,/?%#[\]]+)>;\s*rel="+([A-z]+)"+/
+
+export function parseLinkHeader(header) {
+  const linkCount = (header.match(/</g) || []).length
+  const groups = []
+  for (let i = 0; i < linkCount; i++) groups.push(linkGroup.source)
+  const regex = new RegExp(groups.join(/\s*,\s*/.source))
+  const match = header.match(regex)
+  const links = {}
+  if (match) {
+    for (let i = 1; i < match.length; i = i + 2) {
+      const link = match[i]
+      const rel = match[i + 1]
+      if (!Object.hasOwn(links, rel)) links[rel] = []
+      links[rel].push(link)
+    }
+  }
+  return links
+}
+
```

Comment:
https://www.npmjs.com/package/parse-link-header seems like a pretty reasonable library.  It handles a bunch of edge cases like the `rel` value not requiring quotes or having multiple values.

![screen-rbiop image](https://user-images.githubusercontent.com/350812/206024667-d62dbc2b-73fb-417a-a122-f60f5e78d2bd.png)

https://www.rfc-editor.org/rfc/rfc8288#section-3

Reply:
I tried this but it's a node library and only works in nodejs AFAICT. Someone wrote up an issue [here](https://github.com/thlorenz/parse-link-header/issues/26).

Reply:
:sigh: _tempted to submit a PR to remove the reliance on `process.env`_.  We will just have to be careful when we are serializing `Link` headers.  Since we are using dave-shawley/ietfparse to serialize them, we are safe.  It always inserts quotes and does not handle the multiple `rel` case today.